django_test_suite.sh

```diff
@@ -42,21 +63,11 @@ PASSWORD_HASHERS = [
 !
 }
 
-setup_emulator_if_needed() {
-    if [[ $SPANNER_EMULATOR_HOST != "" ]]
-    then
-        echo "Running the emulator at: $SPANNER_EMULATOR_HOST"
-        ./emulator_main --host_port "$SPANNER_EMULATOR_HOST" &
-        SPANNER_INSTANCE=$INSTANCE python3 .kokoro/ensure_instance_exists.py
-    fi
-}
-
 run_django_tests() {
     cd $TESTS_DIR/django/tests
     create_settings
     echo -e "\033[32mRunning Django tests: $TEST_APPS\033[00m"
-    python3 runtests.py $TEST_APPS --verbosity=2 --noinput --settings $SETTINGS_FILE
+    python3 runtests.py $TEST_APPS --verbosity=3 --noinput --settings $SETTINGS_FILE
```

Comment:
I've moved a part of the `build.sh` to this file to reduce the number of scripts calling each other

Reply:
For debugging only. There was no traceback at all, it was just showing `no Google credentials`, but in fact there was another error. So, yeah, after https://github.com/googleapis/python-spanner/pull/177 fix I'm gonna cleanup this PR once again.